ember_debug/object-inspector.js

```diff
@@ -220,7 +220,7 @@ export default EmberObject.extend(PortMixin, {
   },
 
   canSend(val) {
-    return (val instanceof EmberObject) || typeOf(val) === 'array';
+    return val && ((val instanceof Object) || typeOf(val) === 'array');
```

Comment:
Should we check both `Object` and `EmberObject` to account for everything?

Reply:
done, also pushed an update the better check if `class extends` or `Ember.*.extend` was used by checking if protoype has `_super`